### ISSUE TITLE
Further actions when fine tuning results are very poor

### ISSUE THREAD

**Issue by yspaik** [IDEACVR/DINO#32]
First thank you for your great contribution.

I am trying to train my custom dataset, pretraining with the coco dataset
With your suggestion in the issue, I added parameters. Rest of them was system default.
```
-- pretrain_model_path /path/to/checkpoint/of/coco-pretrained-model
-- finetune_ignore label_enc.weight class_embed
-- options num_classes=7
```

Training proceeds, but performance has become poor. Growth stops around 0.3 AP.
<img width="310" alt="스크린샷 2022-07-22 오전 11 22 03" src="https://user-images.githubusercontent.com/15726007/180349180-88075708-3896-461b-acd6-c3c162d559ea.png">

<img width="1059" alt="스크린샷 2022-07-22 오전 11 19 50" src="https://user-images.githubusercontent.com/15726007/180348669-c0bb392f-e145-46e5-86cd-39bcd03735fe.png">
Blue is the graph obtained at the time of pretraining, and the coco-dataset was trained with dino and backbone as swin large, and the red graph is the fine-tuning of this coco-dino-swin model as pretrained.

<img width="1081" alt="스크린샷 2022-07-22 오전 11 26 48" src="https://user-images.githubusercontent.com/15726007/180349383-1bc66e09-c0ca-420c-87dd-b8b267ac4305.png">

Looking a little more at the loss graph, it was observed that the box loss fell enough as expected, but the class embedding part, where the weight was ignored during finetune, did not fall enough.

What additional parameters do I need to adjust to improve the performance of the model I fine-tune with my own dataset? Or what additional actions should be observed/taken further? Your suggestion will be very appreciated.

**Comment by SuperHenry2333** [comment]
Hi, do you mean you do not train class embedding during finetuning? If yes, I think this is problematic.

**Comment by yspaik** [comment]
Not really. When training my own dataset during fine tuning, both bbox and class embeddings are trained as you response at the issues 23(https://github.com/IDEACVR/DINO/issues/23). However, since ```num_classes``` are different, add ```label_enc.weight class_embed``` to the ```-- finetune_ignore``` option. If I do not add this option, ```num_classes``` mismatch  cause a tensor size mismatch error occurs.

I expected the same level of performance as DINO showed with the coco dataset. I succeeded score in reproducing the performance using dino with the coco dataset as you saw the tensorboard graph above, but the problem is that the performance does not come out when I fine-tune train my own dataset using the coco dataset-swin backbone-dino model as a pretrained model. I would like to know if there are any additional parameters I need to adjust.

**Comment by SuperHenry2333** [comment]
Not sure what causes the problem. Maybe you can try not to change anything such as num_classes and finetune_ignore  and to directly load coco pre-trained models.  Since you have 7 classes, the model will use first 7 class embeddings and leave other 91-7 classes unused.

**Comment by morgankohler** [comment]
How was this issue handled during the fine-tuning of object-365 to coco?
